deployments/local/scripts/deploy.groovy

```diff
@@ -23,7 +23,17 @@ def header(message) {
     println ""
 }
 
+def createLocalS3Store() {
+    header("Deleting contents of .localstack")
+    def tmpDir = new File("../../.localstack")
+    if (tmpDir.exists()) {
+        tmpDir.deleteDir()
+    }
+    tmpDir.mkdir()
+}
+
 header("Deploying the application on " + STACK)
+createLocalS3Store()
 run("docker stack rm " + STACK)
 run("docker network create --driver=overlay --attachable " + NETWORK)
 run("docker stack deploy -c ../../docker-compose.yml -c docker-override.yml " + STACK)
```

Comment:
`run("docker stack deploy -c ../../../docker-compose.yml -c ../docker-override.yml " + STACK)
`

Reply:
I fixed this in my last force push. Now we run a check to make sure you run the script in `scripts` directory